package.json

```diff
@@ -28,26 +28,26 @@
   "author": "Anthony Frehner",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
+    "@babel/preset-react": "^7.9.4",
     "@wext/shipit": "^0.2.1",
-    "babel-loader": "^8.0.6",
+    "babel-loader": "^8.1.0",
     "clean-webpack-plugin": "^3.0.0",
-    "concurrently": "^4.1.1",
-    "copy-webpack-plugin": "^5.0.4",
-    "husky": "^3.0.2",
-    "prettier": "1.18.2",
```

Comment:
you'll probably want to run prettier on the whole repo as a part of this, otherwise subsequent PRs will be sprinkled with actual changes and prettier changes

Reply:
Good call!